src/utils/color-set.ts

```diff
@@ -0,0 +1,8 @@
+import { ColorSetGroup } from './color-set-group.enum';
+
+export interface ColorSet {
+  name: string;
+  selectable: boolean;
+  group: ColorSetGroup;
+  domain: string[];
```

Comment:
There's places where it's being used as a `number[]`, but so far, `colorSets` haven't made use of a single `number[]`. Perhaps I am missing something?